include/lsst/jointcal/Gtransfo.h

```diff
@@ -1,6 +1,5 @@
 // -*- LSST-C++ -*-
 
-/// \file gtransfo.h
 /// \brief Geometrical transformations (of 2D points)
```

Comment:
Don't leave this a Doxygen comment; it will attach to the wrong thing (likely `namespace pexExcept` or  `namespace lsst`).